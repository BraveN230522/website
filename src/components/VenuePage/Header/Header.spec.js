import React from 'react';
import { render } from '@testing-library/react';
import withGlobals from '@swiamTestUtils/withGlobals';
import VenuePageHeader from './index';

const mockData = {
  fromDate: '2020-05-06',
  toDate: '2021-04-11',
  handleChangeDates: () => {},
  data: {
    title: 'Arthur Ashe Stadium',
    description:
      'With a capacity of 23,700 and a cruel nosebleed section, this is a vacuous venue from which fans predictably can sometimes feel detached from the players and excitement. Early Ashe matches usually feature a star against a lowly ranked player or no-name, like #1 Novak Djokovich against an unknown qualifier. Unless you have seats up close or simply must see your favorite player, consider a smaller venue here as there are heaps on offer like below.Louis Armstrong Stadium holds about 10,000 and can be a much better location to more closely view name players, without binoculars. With 5,800 seats, Grandstand stadium makes it possible to intimately experience the intense action. Nearly all seats are available with a Mini-Plan package, including some that are literally first-row courtside. Another bonus: SHADE! It\'s estimated that 20 percent of the seats at Grandstand are in precious, rare shade — practically the ONLY shade in the entire tennis center. Fans often opt for the shady east side of the stadium, which is excellent for visibility and comfort in that there are individual tapered seats. In some areas of Grandstand, there are only butt-flattening bleachers with no back support. Still, here you can easily see and feel the pounding, the pace, and the emotions from the Grandstand seats.In addition to the three major stadiums, there are 16 "outer” courts, where seating capacity ranges from about 100 to 1500. These, too, are intimate spots that are especially inviting during the evening hours when the intense sun had faded. Again, the earlier you arrive, the better your odds of getting a great seat. In some cases fans are denied entry to matches because they arrive late. Lots of doubles matches on the outer courts too. Practice courts are accessible all the time and fans can find plenty of action elsewhere.Fan Author: Chris Martin / Photo credit 32',
    imageURL: 'https://sportswhereiam.com/wp-content/uploads/2015/07/Arthur-Ashe-Stadium.jpg',
    mobileImageURL: 'https://sportswhereiam.com/wp-content/uploads/2015/07/Arthur-Ashe-Stadium.jpg',
  },
};
/**
 * TODO: When graphql request is configured,
 * this toHaveContent tests will be changed for a mock response
 */
describe('<VenuePageHeader />', () => {
  it('Renders a VenuePageHeader component', () => {
    const { fromDate, toDate, data, handleChangeDates } = mockData;
    const { container } = render(
      <VenuePageHeader fromDate={fromDate} toDate={toDate} data={data} handleChangeDates={handleChangeDates} />,
      {
        wrapper: withGlobals,
      }
    );
    expect(container).toMatchSnapshot();
  });
});
