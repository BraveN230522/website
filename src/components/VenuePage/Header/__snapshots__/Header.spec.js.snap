// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VenuePageHeader /> Renders a VenuePageHeader component 1`] = `
.emotion-60 {
  width: 100%;
  min-height: 736px;
  padding-top: 1px;
  background-color: #000000;
  position: relative;
}

.emotion-60:after,
.emotion-60:before {
  content: '';
  display: block;
  position: absolute;
  left: 0;
  top: 0;
  height: 100%;
  width: 100%;
  z-index: 2;
}

.emotion-60:after {
  display: block;
  background: rgba(0,0,0,0.1);
}

.emotion-60:before {
  background: linear-gradient(transparent,70%,#000000 100%);
}

@media (max-width:768px) {
  .emotion-60 {
    min-height: 516px;
  }
}

.emotion-60 figure {
  margin: 0;
}

.emotion-0 {
  position: absolute;
  width: 100%;
  height: 100%;
  top: 0;
  left: 0;
  z-index: 1;
  object-fit: cover;
  background-repeat: no-repeat;
  -webkit-transition: all ease-out .3s;
  transition: all ease-out .3s;
  object-position: center bottom;
  opacity: 1;
}

@media (max-width:48em) {
  .emotion-0 {
    display: none;
  }
}

.emotion-2 {
  position: absolute;
  width: 100%;
  height: 100%;
  top: 0;
  left: 0;
  z-index: 1;
  object-fit: cover;
  background-repeat: no-repeat;
  -webkit-transition: all ease-out .3s;
  transition: all ease-out .3s;
  display: none;
  opacity: 1;
}

@media (max-width:48em) {
  .emotion-2 {
    display: block;
    object-position: center bottom;
    object-fit: cover;
  }
}

@media (max-width:48em) {
  .emotion-2 {
    object-position: center 0;
  }
}

.emotion-10 {
  max-width: 1323px;
  width: 100%;
  margin: 0 auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 46px 20px 0;
  margin-bottom: 38px;
  position: relative;
  z-index: 3;
}

.emotion-10 h1 {
  margin-top: 0;
  margin-bottom: 0;
}

.emotion-4 {
  color: #ffffff;
  cursor: pointer;
  font-size: 16px;
  font-weight: 500;
  line-height: 21px;
  -webkit-letter-spacing: -0.08px;
  -moz-letter-spacing: -0.08px;
  -ms-letter-spacing: -0.08px;
  letter-spacing: -0.08px;
  vertical-align: middle;
}

.emotion-8 {
  list-style: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-6 {
  margin-left: 21px;
  cursor: pointer;
}

.emotion-6:nth-of-type(1) img {
  margin-top: 3px;
  -webkit-transform: scale(1.2);
  -ms-transform: scale(1.2);
  transform: scale(1.2);
}

.emotion-6:last-child {
  display: none;
}

.emotion-6 img {
  -webkit-transition: ease-out 0.3s -webkit-transform;
  -webkit-transition: ease-out 0.3s transform;
  transition: ease-out 0.3s transform;
}

.emotion-6:hover img {
  -webkit-transform: scale(0.9);
  -ms-transform: scale(0.9);
  transform: scale(0.9);
}

@media (max-width:60em) {
  .emotion-6:nth-of-type(2) {
    -webkit-transform: scale(1.3);
    -ms-transform: scale(1.3);
    transform: scale(1.3);
    margin-right: 14px;
    margin-top: 2px;
  }

  .emotion-6:nth-of-type(even) {
    display: none;
  }

  .emotion-6:last-child {
    display: block;
  }
}

.emotion-28 {
  position: relative;
  z-index: 4;
}

.emotion-26 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  max-width: 1255px;
  margin: 0 auto;
  padding: 0 2rem;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  margin-bottom: 106px;
}

@media (max-width:48em) {
  .emotion-26 {
    margin-bottom: 34px;
  }
}

.emotion-14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: baseline;
  -webkit-box-align: baseline;
  -ms-flex-align: baseline;
  align-items: baseline;
}

.emotion-14 svg {
  margin-left: 20px;
}

.emotion-12 {
  font-size: 41px;
  width: 100%;
  font-family: DINAlternateBold,sans-serif;
  color: #ffffff;
  font-stretch: normal;
  line-height: 1.22;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  font-style: normal;
  font-weight: 400;
  margin-bottom: 12px;
}

@media (max-width:48em) {
  .emotion-12 {
    margin: 0 0 16px;
    font-size: 32px;
  }

  .emotion-12 span {
    display: block;
    font-size: 32px;
  }
}

@media (max-width:22.8125em) {
  .emotion-12 {
    font-size: 31px;
  }

  .emotion-12 span {
    font-size: 31px;
  }
}

.emotion-12 span {
  color: #2891ff;
}

.emotion-16 {
  color: #ffffff;
  font-family: Inter,sans-serif;
  font-size: 14px;
  font-weight: 400;
  line-height: 19px;
  padding: 0 166px;
  text-align: center;
}

@media (max-width:48em) {
  .emotion-16 {
    padding: 0;
  }
}

.emotion-23 {
  color: #ffffff;
  font-size: 19px;
  font-weight: 400;
  text-align: center;
  margin: 0;
}

@media (max-width:48em) {
  .emotion-23 {
    font-size: 15px;
    font-weight: 400;
    line-height: 20px;
  }
}

.emotion-58 {
  position: fixed;
  top: 0;
  right: -100rem;
  visibility: hidden;
  opacity: 0;
  width: 100%;
  height: 100%;
  z-index: 500;
  -webkit-transition: all 0.32s ease-in-out;
  transition: all 0.32s ease-in-out;
}

.emotion-58:after {
  content: '';
  background-color: rgba(0,0,0,0.7);
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
}

.emotion-56 {
  background-color: #ffffff;
  position: absolute;
  width: 80%;
  height: 100%;
  top: 0;
  right: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  z-index: 502;
  padding-bottom: 24px;
  border-radius: 12px 0 0 12px;
}

@media (max-width:48em) {
  .emotion-56 {
    max-width: 338px;
  }
}

.emotion-32 {
  background-color: #2891ff;
  min-height: 93px;
  padding: 12px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  border-top-left-radius: 12px;
  -webkit-letter-spacing: -0.17px;
  -moz-letter-spacing: -0.17px;
  -ms-letter-spacing: -0.17px;
  letter-spacing: -0.17px;
  color: #ffffff;
  position: relative;
}

.emotion-32 > span {
  font-family: DINAlternateRegular,sans-serif;
  font-size: 24px;
}

.emotion-30 {
  border: 0;
  background-color: transparent;
  width: 40px;
  height: 40px;
  border-radius: 50%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: absolute;
  left: -41px;
  top: 57px;
  outline: 0;
}

.emotion-30:focus {
  outline: 0;
}

.emotion-54 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 12px;
  text-align: right;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  height: 100%;
  overflow-y: auto;
}

.emotion-34 {
  font-family: DINAlternateBold,sans-serif;
  font-size: 24px;
  margin-top: 24px;
  margin-bottom: 24px;
  color: #000000;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-34:hover,
.emotion-34:visited,
.emotion-34:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-34 > svg {
  margin-left: 25px;
}

.emotion-48 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-bottom: 60px;
}

.emotion-36 {
  font-family: DINAlternateBold,sans-serif;
  font-size: 24px;
  margin-top: 24px;
  color: #000000;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-36:hover,
.emotion-36:visited,
.emotion-36:focus {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-36 > svg {
  margin-left: 25px;
}

.emotion-19 {
  position: relative;
  padding-left: 16px;
  min-width: 100px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0;
  text-align: left;
  padding-left: 0;
  margin: 0;
}

.emotion-19 .DateRangePicker.DateRangePicker_1 {
  width: 100%;
}

@media (max-width:48em) {
  .emotion-19 {
    padding-right: 16px;
    min-width: 80px;
  }
}

@media (max-width:48em) {
  .emotion-19 {
    padding-right: 0;
  }
}

.emotion-19 .DateRangePickerInput_calendarIcon {
  margin: 0;
  padding: 0;
  display: inline-block;
  width: 34px;
}

.emotion-19 .DayPicker_transitionContainer {
  position: relative;
  overflow: hidden;
  border-radius: 3px;
  max-width: 560px;
  max-height: 300px;
  background: transparent;
  border-radius: 16px;
}

.emotion-19 .DateRangePicker_picker.DateRangePicker_picker_1 {
  right: -85px !important;
}

@media (max-width:48em) {
  .emotion-19 .DateRangePicker_picker.DateRangePicker_picker_1 {
    right: -25px !important;
  }
}

.emotion-19 .DateRangePicker {
  border: 2px solid #ffffff;
  border-radius: 25px;
  width: 320px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 45px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding-left: 12px;
}

@media (max-width:48em) {
  .emotion-19 .DateRangePicker {
    padding-left: 0;
  }
}

.emotion-19 .DateRangePicker input {
  text-align: center;
  padding-top: 4px;
  background-color: transparent;
}

.emotion-19 .DateRangePicker:hover {
  background-color: #ffffff;
}

.emotion-19 .DateRangePicker:hover button > svg > g > path {
  stroke: #000000;
}

.emotion-19 .DateRangePicker:hover #startDateInputId,
.emotion-19 .DateRangePicker:hover #endDateInputId {
  color: #000000;
}

.emotion-19 .DateRangePicker:hover #startDateInputId::-webkit-input-placeholder,
.emotion-19 .DateRangePicker:hover #endDateInputId::-webkit-input-placeholder {
  color: #000000;
  opacity: 1;
}

.emotion-19 .DateRangePicker:hover #startDateInputId::-moz-placeholder,
.emotion-19 .DateRangePicker:hover #endDateInputId::-moz-placeholder {
  color: #000000;
  opacity: 1;
}

.emotion-19 .DateRangePicker:hover #startDateInputId:-ms-input-placeholder,
.emotion-19 .DateRangePicker:hover #endDateInputId:-ms-input-placeholder {
  color: #000000;
  opacity: 1;
}

.emotion-19 .DateRangePicker:hover #startDateInputId::placeholder,
.emotion-19 .DateRangePicker:hover #endDateInputId::placeholder {
  color: #000000;
  opacity: 1;
}

.emotion-19 .DateRangePicker:hover .DateRangePickerInput_arrow.DateRangePickerInput_arrow_1 {
  color: #000000;
}

.emotion-19 .DateRangePickerInput_arrow.DateRangePickerInput_arrow_1 {
  font-weight: 600;
}

.emotion-19 .DateRangePicker_picker {
  background-color: transparent;
}

.emotion-19 .DateRangePickerInput {
  border: none;
  background: transparent;
}

.emotion-19 #startDateInputId {
  width: 120px;
  margin-right: 0px;
}

.emotion-19 #endDateInputId {
  width: 120px;
  margin-left: 0px;
}

.emotion-19 #startDateInputId,
.emotion-19 #endDateInputId {
  border: none;
  font-size: 16px;
  font-family: Inter,sans-serif;
  color: #ffffff;
  -webkit-letter-spacing: -0.11px;
  -moz-letter-spacing: -0.11px;
  -ms-letter-spacing: -0.11px;
  letter-spacing: -0.11px;
  font-weight: 600;
}

.emotion-19 #startDateInputId::-webkit-input-placeholder,
.emotion-19 #endDateInputId::-webkit-input-placeholder {
  font-size: 16px;
  font-family: Inter,sans-serif;
  color: #ffffff;
  -webkit-letter-spacing: -0.11px;
  -moz-letter-spacing: -0.11px;
  -ms-letter-spacing: -0.11px;
  letter-spacing: -0.11px;
  opacity: 1;
}

.emotion-19 #startDateInputId::-moz-placeholder,
.emotion-19 #endDateInputId::-moz-placeholder {
  font-size: 16px;
  font-family: Inter,sans-serif;
  color: #ffffff;
  -webkit-letter-spacing: -0.11px;
  -moz-letter-spacing: -0.11px;
  -ms-letter-spacing: -0.11px;
  letter-spacing: -0.11px;
  opacity: 1;
}

.emotion-19 #startDateInputId:-ms-input-placeholder,
.emotion-19 #endDateInputId:-ms-input-placeholder {
  font-size: 16px;
  font-family: Inter,sans-serif;
  color: #ffffff;
  -webkit-letter-spacing: -0.11px;
  -moz-letter-spacing: -0.11px;
  -ms-letter-spacing: -0.11px;
  letter-spacing: -0.11px;
  opacity: 1;
}

.emotion-19 #startDateInputId::placeholder,
.emotion-19 #endDateInputId::placeholder {
  font-size: 16px;
  font-family: Inter,sans-serif;
  color: #ffffff;
  -webkit-letter-spacing: -0.11px;
  -moz-letter-spacing: -0.11px;
  -ms-letter-spacing: -0.11px;
  letter-spacing: -0.11px;
  opacity: 1;
}

.emotion-19 #startDateInputId:focus,
.emotion-19 #endDateInputId:focus {
  box-shadow: 0 4px 2px -2px #2891ff;
}

.emotion-19 .DateInput {
  background-color: transparent;
  width: 42%;
}

@media (max-width:48em) {
  .emotion-19 .DateInput {
    display: none;
  }
}

.emotion-19 .DateRangePickerInput_arrow {
  color: #ffffff;
}

@media (max-width:48em) {
  .emotion-19 .DateRangePickerInput_arrow {
    display: none;
  }
}

.emotion-19 .DateInput_fang.DateInput_fang_1 {
  display: none;
}

.emotion-19 .DayPicker {
  border: 2px solid #d1d1d1;
  border-radius: 16px;
  box-sizing: border-box;
}

.emotion-19 .DayPicker::before {
  content: '';
  position: absolute;
  left: calc(50% + 3px);
  top: -1px;
  z-index: 2;
  width: 2px;
  height: calc(100% + 2px);
  background-color: #d1d1d1;
}

@media (max-width:48em) {
  .emotion-19 .DayPicker::before {
    display: none;
  }
}

.emotion-19 .CalendarMonth_caption {
  padding-top: 12px;
  font-family: Inter,sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 13px;
  line-height: 24px;
  text-align: center;
  color: #565656;
}

.emotion-19 .CalendarMonth_caption strong {
  font-weight: 500;
}

.emotion-19 .DayPicker_weekHeader {
  top: 46px;
}

.emotion-19 .DayPicker_weekHeader_li {
  visibility: hidden;
  font-size: 0;
}

.emotion-19 .DayPicker_weekHeader_li::first-letter {
  visibility: visible;
  font-family: Inter,sans-serif;
  font-style: normal;
  font-weight: 500;
  font-size: 12px;
  line-height: 14px;
  text-align: center;
  color: rgba(0,0,0,0.38);
}

.emotion-19 .CalendarMonth_table.CalendarMonth_table_1 {
  margin-top: 0px;
}

.emotion-19 .CalendarDay {
  font-family: Inter,sans-serif;
  font-size: 12px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  text-align: center;
}

.emotion-19 .CalendarDay__selected_span {
  background: rgba(40,145,255,0.2);
  color: #ffffff;
}

.emotion-19 .CalendarDay__selected {
  background: #2891ff;
  color: #ffffff;
}

.emotion-19 .CalendarDay__selected:hover {
  background: #2891ff;
  color: #ffffff;
}

.emotion-19 .CalendarDay__hovered_span:hover,
.emotion-19 .CalendarDay__hovered_span {
  background: rgba(40,145,255,0.2);
}

.emotion-19 .CalendarDay__default {
  border: none;
  color: #000000;
}

.emotion-19 table {
  border-spacing: 0 3px;
  border-collapse: separate;
}

.emotion-19 .CalendarDay.CalendarDay__firstDayOfWeek {
  border-top-left-radius: 50%;
  border-bottom-left-radius: 50%;
}

.emotion-19 .CalendarDay.CalendarDay__lastDayOfWeek {
  border-top-right-radius: 50%;
  border-bottom-right-radius: 50%;
}

.emotion-19 .CalendarDay.CalendarDay__selected_start.CalendarDay__selected,
.emotion-19 .CalendarDay.CalendarDay__selected_end.CalendarDay__selected {
  border-radius: 50%;
  color: #ffffff;
  position: relative;
}

.emotion-19 .CalendarDay.CalendarDay__selected_start.CalendarDay__selected::before,
.emotion-19 .CalendarDay.CalendarDay__selected_end.CalendarDay__selected::before {
  content: '';
  position: absolute;
  top: 0px;
  bottom: 0px;
  z-index: 2;
  width: 10px;
  background-color: rgba(40,145,255,0.2);
}

.emotion-19 .CalendarDay.CalendarDay__selected_end.CalendarDay__selected::before {
  left: 0px;
}

.emotion-19 .CalendarDay.CalendarDay__selected_start.CalendarDay__selected::before {
  right: 0px;
}

.emotion-19 .CalendarDay__blocked_out_of_range,
.emotion-19 .CalendarDay__blocked_out_of_range:active,
.emotion-19 .CalendarDay__blocked_out_of_range:hover {
  color: #d1d1d1;
  cursor: not-allowed;
}

.emotion-19 .CalendarDay__selected_span {
  border-top-left-radius: 50%;
  border-bottom-left-radius: 50%;
}

.emotion-19 .CalendarDay__selected_span.__last-day-of-month__ {
  border-top-right-radius: 50%;
  border-bottom-right-radius: 50%;
}

.emotion-19 .CalendarDay__selected_span ~ .CalendarDay__selected_span,
.emotion-19 .CalendarDay__selected_start ~ .CalendarDay__selected_span {
  border-top-left-radius: 0;
  border-bottom-left-radius: 0;
}

.emotion-19 .DateRangePicker.DateRangePicker_1 {
  max-width: 339px;
}

<div>
  <header
    class="emotion-60 emotion-61"
  >
    <figure>
      <div>
        <img
          class="emotion-0 emotion-1"
          src="https://sportswhereiam.com/wp-content/uploads/2015/07/Arthur-Ashe-Stadium.jpg"
        />
        <img
          class="emotion-2 emotion-3"
          src="https://sportswhereiam.com/wp-content/uploads/2015/07/Arthur-Ashe-Stadium.jpg"
        />
      </div>
    </figure>
    <div
      class="emotion-10 emotion-11"
    >
      <h1>
        <a
          class="emotion-4 emotion-5"
        />
      </h1>
      <nav>
        <ul
          class="emotion-8 emotion-9"
        >
          <li
            class="emotion-6 emotion-7"
          >
            <a>
              <img
                alt="bars"
                src="mocked-file"
              />
            </a>
          </li>
        </ul>
      </nav>
    </div>
    <div
      class="emotion-28 emotion-29"
    >
      <div
        class="emotion-25 emotion-26 emotion-27"
        direction="center"
      >
        <div>
          <div
            class="emotion-14 emotion-15"
          >
            <h2
              class="emotion-12 emotion-13"
            >
              Arthur Ashe Stadium
            </h2>
          </div>
          <p
            class="emotion-16 emotion-17"
          />
          <div
            class="emotion-18 emotion-19 emotion-20"
          >
            MOCKED RANGE PICKER
          </div>
          <p
            class="emotion-16 emotion-17"
          >
            With a capacity of 23,700 and a cruel nosebleed section, this is a vacuous venue from which fans predictably can sometimes feel detached from the players and excitement. Early Ashe matches usually feature a star against a lowly ranked player or no-name, like #1 Novak Djokovich against an unknown qua
          </p>
          <h3
            class="emotion-23 emotion-24"
          />
        </div>
      </div>
    </div>
    <div
      aria-hidden="true"
      class="emotion-58 emotion-59"
      data-testid="menu-mobile"
    >
      <aside
        class="emotion-56 emotion-57"
      >
        <div
          class="emotion-32 emotion-33"
        >
          <span>
            Menu
          </span>
          <button
            class="emotion-30 emotion-31"
          >
            <svg
              color="#fff"
              height="24"
              width="24"
            >
              <defs>
                <path
                  d="M.293.293c.39-.39 1.024-.39 1.414 0L7 5.585 12.293.293c.36-.36.928-.388 1.32-.083l.094.083c.39.39.39 1.024 0 1.414L8.415 7l5.292 5.293c.36.36.388.928.083 1.32l-.083.094c-.39.39-1.024.39-1.414 0L7 8.415l-5.293 5.292c-.36.36-.928.388-1.32.083l-.094-.083c-.39-.39-.39-1.024 0-1.414L5.585 7 .293 1.707C-.067 1.347-.095.78.21.387z"
                  id="prefix__a"
                />
              </defs>
              <g
                fill="none"
                fill-rule="evenodd"
                transform="translate(5 5)"
              >
                <mask
                  fill="#fff"
                  id="prefix__b"
                >
                  <use
                    xlink:href="#prefix__a"
                  />
                </mask>
                <use
                  fill="#fff"
                  fill-rule="nonzero"
                  xlink:href="#prefix__a"
                />
                <g
                  fill="#fff"
                  mask="url(#prefix__b)"
                >
                  <path
                    d="M0 0H24V24H0z"
                    transform="translate(-5 -5)"
                  />
                </g>
              </g>
            </svg>
          </button>
        </div>
        <div
          class="emotion-54 emotion-55"
        >
          <a
            class="emotion-34 emotion-35"
            href="/"
          >
            Login / Register 
            <svg
              color="#000"
              fill="#000"
              height="24"
              viewBox="0 0 19.001 24"
              width="19.001"
            >
              <g
                transform="translate(-0.999)"
              >
                <path
                  d="M5,12c2.761,0,5-2.686,5-6S9.265,0,5,0,0,2.686,0,6,2.239,12,5,12Z"
                  transform="translate(5)"
                />
                <path
                  d="M.051.1c0-.2,0-.056,0,0Z"
                  transform="translate(0.95 20.949)"
                />
                <path
                  d="M.049.18c0-.054,0-.375,0,0Z"
                  transform="translate(18.95 20.91)"
                />
                <path
                  d="M18.989,8.405C18.9,2.669,18.129,1.035,12.254,0A3.558,3.558,0,0,1,9.5,1.028,3.559,3.559,0,0,1,6.746,0C.936,1.023.121,2.634.014,8.219.006,8.675,0,8.7,0,8.646c0,.1,0,.283,0,.6C0,9.249,1.4,12,9.5,12S19,9.249,19,9.249c0-.206,0-.349,0-.446A3.6,3.6,0,0,1,18.989,8.405Z"
                  transform="translate(1 12)"
                />
              </g>
            </svg>
          </a>
          <nav
            class="emotion-48 emotion-49"
          >
            <a
              class="emotion-36 emotion-35"
              href="/"
            >
              Home
            </a>
            <a
              class="emotion-36 emotion-35"
              href="/"
            >
              Events
            </a>
            <a
              class="emotion-36 emotion-35"
              href="/"
            >
              Cities
            </a>
            <a
              class="emotion-36 emotion-35"
              href="/"
            >
              Hotels
            </a>
            <a
              class="emotion-36 emotion-35"
              href="/"
            >
              Experiences
            </a>
            <a
              class="emotion-36 emotion-35"
              href="/"
            >
              About
            </a>
          </nav>
          <a
            class="emotion-36 emotion-35"
            href="/"
          >
            Contact
          </a>
          <a
            class="emotion-36 emotion-35"
            href="/"
          >
            Download App for iOS
          </a>
        </div>
      </aside>
    </div>
  </header>
</div>
`;
